open-craft/opencraft
# [BB-2067] Remove Gandi v4 code

This PR removes all the code corresponding to the Gandi v4 API. It also fixes an issue causing the DNS records cleanup in the cleanup job to fail.

**Testing instructions**:
* Verify that the CI checks pass.
* Verify that this doesn't break anything else.
